styles/components/layout/Header.module.sass

```
@@ -1,3 +1,46 @@
 @use '@/styles/utils/mixins' as m
 @use '@/styles/utils/funcs' as f
-@use '@/styles/themes/default/Colors.module.sass' as c
\ No newline at end of file
+@use '@/styles/themes/default/Colors.module.sass' as c
+@use 'sass:math'
+
+
+$logo_m: 9.4%
+
+
+.container
+
+.wrapper
+    display: flex
+    align-items: center
+    @include m.media(phone, tablet)
+        padding-top: f.px(56) 
+        padding-bottom: f.px(56) 
+    @include m.media(laptop, desktop)
+        padding-top: f.px(20) 
+        padding-bottom: f.px(20) 
+
+.iconLogo
+    margin-right: $logo_m
+
+.logos 
+    display: flex
+    align-items: center
+    flex-basis: math.div(36%, 1)
```

no need to divide by one

```suggestion
    flex-basis: 36%
```